lib/a11y-snapshot/screen-reader.test.js

```diff
@@ -222,4 +222,53 @@ describe("nvda", () => {
 			}).toMatchSpeechInlineSnapshot(`"0°C"`);
 		}, 30000);
 	});
+
+	describe("TimePicker", () => {
+		test("keyboard navigation for desktop variant", async () => {
+			const page = await browser.newPage();
+			await blockAds(page);
+			await gotoMuiPage(page, "/components/time-picker");
+
+			const [desktopTimePickerOpenTrigger] = await page.$$(
+				'#ResponsiveTimePickers + div button[aria-label*="Choose time"]'
+			);
+
+			await expect(() =>
+				desktopTimePickerOpenTrigger.focus()
+			).toMatchSpeechInlineSnapshot(`"clickable, Choose tim, button"`);
+
+			await expect(() => page.keyboard.press("Enter"))
+				.toMatchSpeechInlineSnapshot(`
+						"dialog"
+						"dialog"
+						"Select hours. Selected time is 1:00 AM, list"
+					`);
+
+			await expect(() =>
+				page.keyboard.press("ArrowUp")
+			).toMatchSpeechInlineSnapshot(`"Select hours. Selected time is 2:00 AM"`);
+
+			await expect(() =>
+				page.keyboard.press("ArrowDown")
+			).toMatchSpeechInlineSnapshot(`"Select hours. Selected time is 1:00 AM"`);
+
+			await expect(() =>
+				page.keyboard.press("End")
+			).toMatchSpeechInlineSnapshot(
+				`"Select hours. Selected time is 11:00 AM"`
+			);
+
+			await expect(() =>
+				page.keyboard.press("Home")
+			).toMatchSpeechInlineSnapshot(
+				`"Select hours. Selected time is 12:00 AM"`
+			);
+
+			await expect(() =>
+				page.keyboard.press("Escape")
+			).toMatchSpeechInlineSnapshot(
+				`"clickable, main landmark, button, Choose tim"`
```

Comment:
Bug in `screen-reader-testing-library` that should be fixed with https://github.com/eps1lon/screen-reader-testing-library/pull/26